### ISSUE TITLE
Trying to update my videopaks to videolab 1.1.1

### ISSUE THREAD

**Issue by cemillerena** [teenageengineering/videolab#34]
I have created about a dozen videopaks for the previous videolab versions, but none of them are compatible with videolab 1.1.1

I have tried updating them with no luck. Building them on the correct Unity version, made sure they are using OpenGLES2. Getting no errors when building them, but when uploading them to the OP-Z app and trying to open them, they just crash the app and it closes. 

Is there any incompatibility between videolab versions we should be aware of? Kind of a bum that all of my videopaks are no longer working...

**Comment by finnTE** [comment]
Could you post a videopak here, I can have a look.

**Comment by cemillerena** [comment]
> Could you post a videopak here, I can have a look.

**Comment by cemillerena** [comment]
I'm attaching the builds... or do you need the whole Unity project?

[eva01.zip](https://github.com/teenageengineering/videolab/files/4306471/eva01.zip)

**Comment by finnTE** [comment]
can check the build first

**Comment by cemillerena** [comment]
https://drive.google.com/drive/folders/1ICK2isjCsvnGlqe27pk2SgXT_pQvT8gR

That’s a link to the Unity project.

**Comment by cemillerena** [comment]
Hi @finnTE, any news? Since we are all quarantined, I might as well try and fix all of my videopaks, but dont know where to start...

**Comment by finnTE** [comment]
Sorry to keep you waiting, we are a bit swamped atm.

I compiled keijiros VideolabTest a while ago. I cant imagine what your problems are, also I imported videolab from the released package.

https://github.com/teenageengineering/VideolabTest/releases

**Comment by finnTE** [comment]
I run 2018.4.12 btw, __should__ not make a difference.

**Comment by finnTE** [comment]
added some missing info here:
https://github.com/teenageengineering/videolab/wiki/Exporting-for-OP%E2%80%90Z

**Comment by cemillerena** [comment]
@finnTE hey! I actually tried  to build with 2018.4.12 and it did work... same settings as when I was building from 2018.4.18. 

Maybe recommend building from 4.12

**Comment by finnTE** [comment]
Damnit Unity. Will update the dox.

**Comment by cemillerena** [comment]
@finnTE yeah... it's a lot of trial and error. Finally had time today to figure out what was actually happening. Apparently it's a combination of the Unity version but also some patcher parameters I had going on to control camera fx (videolab effects) are not supported or have different values. Some                   of my videopaks are working straight out of 4.12 but others seem to need some meddling in the patcher.
